docs/learn/learn-nomination-pools.md

```diff
@@ -0,0 +1,143 @@
+---
+id: learn-nomination-pools
+title: Nomination Pools
+sidebar_label: Nomination Pools
+description: Learn about Nomination Pools and their features
+keyword: [nominate, nominator, stake, staking, pools]
+slug: ../learn-nomination-pools
+---
+
+[Nomination pools](https://github.com/paritytech/substrate/pull/10694) are one of the key
+(expertimental) features from the roadmap of
+[Staking improvements](https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376) on
+Polkadot. They are designed to permissionlessly allow users (delegators) to pool their funds
```

Comment:
Let's not use the term delegator and stick to the generic term member. 

In fact, at the bottom of the doc, we should have something like a footnote or FAQ that explains why we made this choice: delegator is associated too much with DPoS and since polkadot is NPoS we wanted to stay away. `member` is our generic replacement for delegator. in Action, members are actually quite similar to delegators and do delegate their nomination power to the pool.